### ISSUE TITLE
Not retrieving correct source

### ISSUE THREAD

**Issue by Spidermanvos** [Sklore/HL_DD_5e_Colab#744]
I just tried to load the 5e source on my lap top using "http://www.shadowsoftware.net/herolab/5e/updates.xml" and it loaded the 4e version. Am I the only one having this problem?

**Comment by dungeonguru** [comment]
are you sure you're loading in 5e, and not the 4e authoring kit? I just redownloaded the 5e files and I get what is expected. The initial start screen, make sure you have 5e SRD selected and not the 4e.

**Comment by Spidermanvos** [comment]
How do I make sure I select the 5e are? All I did was enter the url then it automatically grabbed the file and it said 4e. This is the first time this is happened to me, I have loaded the url on my iPad and it works fine. I’m at a loss

Sent from my iPhone

On Oct 14, 2017, at 2:01 PM, dungeonguru <notifications@github.com<mailto:notifications@github.com>> wrote:


are you sure you're loading in 5e, and not the 4e authoring kit? I just redownloaded the 5e files and I get what is expected. The initial start screen, make sure you have 5e SRD selected and not the 4e.
